### PR TITLE
Make tests works with java 10

### PR DESCRIPTION
**Pull request by mouadino**

### CODE REVIEW

File: project.clj [hunk 1]
```diff
@@ -64,4 +65,4 @@
                         :compiler     {:main          com.github.discoverAI.snake.test-runner
                                        :output-to     "resources/public/js/compiled/tests.js"
                                        :output-dir    "resources/public/js/compiled/test/out"
-                                       :optimizations :none}}]})
+                                       :optimizations :none}}]})
```
Comment (mouadino) [inline project.clj]: My editor removes EOL newlines, you can tell me if this fine or not.

**Comment by meandor** [comment]
the test was red :( can you maybe check that again?

File: project.clj (hunk 1 above)
Comment (meandor) [reply project.clj]: looks fine

**Comment by mouadino** [comment]
@meandor Updated the code but it seems that it fails in another place is this by any chance related to an older change?

**Pull request merged**